_base_/docker-compose.yml

```diff
@@ -40,3 +40,15 @@ services:
     command: minio server --quiet --address ${MINIO_ENDPOINT} /data
     networks:
       - aether
+
+  redis:
+    image: redis:alpine
+    command: >
+      redis-server
+      --requirepass ${REDIS_PASSWORD}
+      --notify-keyspace-events KEA
+      --appendonly yes
+      --auto-aof-rewrite-percentage 100
+      --auto-aof-rewrite-min-size 64mb
+    networks:
```

Comment:
Could you add this and check that everything works? (Google disables a couple of commands by default)

```yml
    environment:
      - DISABLE_COMMANDS=FLUSHDB,FLUSHALL
```